Tests/Admin/AdminTest.php

```diff
@@ -22,7 +22,7 @@ class AdminTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
-        $this->contextManager = $this->getMock('Sonata\ClassificationBundle\Model\ContextManagerInterface');
+        $this->contextManager = $this->getMockForAbstractClass('Sonata\ClassificationBundle\Model\ContextManagerInterface');
```

Comment:
We cant use `createMock` because of BC with old PHPUnit versions.
This method is used for interfaces in other sonata bundles. Example from AdminBundle:
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Tests/Admin/BaseFieldDescriptionTest.php#L83